data/autoyast_sle15/autoyast_pcm_allpatterns_ppc64le.xml.ep

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
It seems this profile is very specific for pcm+lgm+ all patterns, but why still need some conditions there, I mean this can be an pure static profile. Anyway, this can work, you can decide it.
One minor comments, the name should be data/autoyast_sle15/autoyast_pcm_lgm_allpatterns_ppc64le.xml.ep

Reply:
Updated the profile name.